### PR TITLE
DOC: use bibtex in plot_simulated_raw_data_using_subject_anatomy.py (#8904)

### PR DESCRIPTION
**Pull request by timonmerk**
#### Reference issue
DOC: #8904

#### What does this implement/fix?
use bibtex in plot_simulated_raw_data_using_subject_anatomy.py

### CODE REVIEW

File: examples/simulation/plot_simulated_raw_data_using_subject_anatomy.py [hunk 1]
```diff
@@ -230,10 +230,9 @@ def data_fun(times, latency, duration):
...
                       hemi='split', views=['lat', 'med'])
+                      
 
 ###############################################################################
 # References
 # ----------
-# .. [1] Destrieux C, Fischl B, Dale A, Halgren E (2010). Automatic
-#        parcellation of human cortical gyri and sulci using standard
-#        anatomical nomenclature, vol. 53(1), 1-15, NeuroImage.
+# .. footbibliography::
```
Comment (timonmerk) [inline examples/simulation/plot_simulated_raw_data_using_subject_anatomy.py:238]:
```suggestion
# .. footbibliography::

```

File: examples/simulation/plot_simulated_raw_data_using_subject_anatomy.py [hunk 2]
```diff
@@ -70,7 +70,7 @@
 # In order to simulate source time courses, labels of desired active regions
 # need to be specified for each of the 4 simulation conditions.
 # Make a dictionary that maps conditions to activation strengths within
-# aparc.a2009s [1]_ labels. In the aparc.a2009s parcellation:
+# aparc.a2009s :footcite:`DestrieuxEtAl2010` labels. In the aparc.a2009s parcellation:
```
Comment (timonmerk) [inline examples/simulation/plot_simulated_raw_data_using_subject_anatomy.py:73]:
```suggestion
# aparc.a2009s :footcite:`DestrieuxEtAl2010` labels. 
# In the aparc.a2009s parcellation:
```

**Review by timonmerk — COMMENTED** [review]
Fix comments

File: examples/simulation/plot_simulated_raw_data_using_subject_anatomy.py (hunk 1 above)
Comment (timonmerk) [inline examples/simulation/plot_simulated_raw_data_using_subject_anatomy.py]:
```suggestion
# .. footbibliography::

```

**Comment by drammock** [comment]
The Style checker is complaining about trailing whitespace:

https://github.com/mne-tools/mne-python/pull/9053/checks?check_run_id=2113273565

**Comment by larsoner** [comment]
Thanks @timonmerk !

**Pull request merged**